alerting/src/test/kotlin/org/opensearch/alerting/AlertingRestTestCase.kt

```diff
@@ -129,19 +130,16 @@ abstract class AlertingRestTestCase : ODFERestTestCase() {
         return response
     }
 
+    /**
+     * Destinations are now deprecated in favor of the Notification plugin's configs.
+     * This method should only be used for checking legacy behavior/Notification migration scenarios.
+     */
     protected fun createDestination(destination: Destination = getTestDestination(), refresh: Boolean = true): Destination {
-        val response = client().makeRequest(
-            "POST",
-            "$DESTINATION_BASE_URI?refresh=$refresh",
-            emptyMap(),
-            destination.toHttpEntity()
-        )
-        assertEquals("Unable to create a new destination", RestStatus.CREATED, response.restStatus())
+        val response = indexDocWithAdminClient(ScheduledJob.SCHEDULED_JOBS_INDEX, UUIDs.base64UUID(), destination.toJsonStringWithType(), refresh)
```

Comment:
Can you add the alert config index mapping before creating the destination and do the same for createEmailAccount and createEmailGroup?

Reply:
Yeah. Also just realized I need to remove the security related tests that aren't relevant anymore as well.